apps/myaccount/src/components/federated-associations/federated-associations.tsx

```diff
@@ -259,7 +261,9 @@ export const FederatedAssociations: FunctionComponent<FederatedAssociationsProps
                                             </Grid.Column>
                                             { !isNonLocalCredentialUser &&
```

Comment:
If we are allowing to delete the IDPs except the one that user signup, do we need to check on whether the user is local or not, it should be common to both right?